lib/matplotlib/tests/test_axes.py

```diff
@@ -3977,6 +3977,17 @@ def test_vlines():
     ax5.set_xlim(0, 15)
 
 
+@check_figures_equal(extensions=["png"])
```

Comment:
A figure comparison is more effort than needed. One could simply create the colors of the returned `LineCollection`.

I'm not insisting on that type of test here, but IMHO it would be more compact and faster. I've not timed figure comparisons, but since the test suite already runs for a long time, it may be worth keeping an eye at adding efficient tests.